tests/core/data/test_excel_data_node.py

```diff
@@ -263,6 +263,44 @@ def test_write(self, excel_file, default_data_frame, content, columns):
         excel_dn.write(None)
         assert len(excel_dn.read()) == 0
 
+    @pytest.mark.parametrize(
+        "content,columns,sheet_name",
+        [
+            ([{"a": 11, "b": 22, "c": 33}, {"a": 44, "b": 55, "c": 66}], None, "sheet_name"),
+            ([[11, 22, 33], [44, 55, 66]], ["e", "f", "g"], "sheet_name"),
+        ],
+    )
+    def test_write_with_sheet_name(self, excel_file_with_sheet_name, default_data_frame, content, columns, sheet_name):
+        if sheet_name:
+            excel_dn = ExcelDataNode(
+                "foo", Scope.PIPELINE, properties={"path": excel_file_with_sheet_name, "sheet_name": sheet_name}
+            )
+        else:
+            excel_dn = ExcelDataNode("foo", Scope.PIPELINE, properties={"path": excel_file_with_sheet_name})
```

Comment:
Useless regarding the set of parameters

Reply:
hmmm I dont think I get what you mean?

Reply:
The else condition is never used. The `sheet_name` variable always equals `"sheet_name"`.